railties/test/application/configuration_test.rb

```diff
@@ -707,6 +707,14 @@ def index
       assert_match(/Missing.*RAILS_MASTER_KEY/, error)
     end
 
+    test "credentials does not raise error when require_master_key is false and master key does not exist" do
```

Comment:
Does this also apply to the case where `require_master_key` isn't specified?

Reply:
Yes, it applies. Behavior when `require_master_key` is not specified is the same as behavior when false is specified.